README.md

```diff
@@ -1,33 +1,33 @@
 
 AnimalTypes
 
-GET http://localhost:9721/api/animals/types
+GET http://localhost:9721/animals/types
```

Comment:
If you are using swagger, why duplicate queries in this file?

Reply:
I copy links from  this file for fast using in Postman

Reply:
Please read about README.md file 

if you whant create file with curls, please use postman file or other

Reply:
done